src/lib/Components/Bidding/Screens/ConfirmFirstTimeBid.tsx

```diff
@@ -79,8 +135,136 @@ export class ConfirmFirstTimeBid extends React.Component<ConfirmBidProps, Confor
     action_type: Schema.ActionTypes.Tap,
     action_name: Schema.ActionNames.BidFlowPlaceBid,
   })
-  placeBid() {
-    return null
+  registerAndPlaceBid() {
+    this.setState({ isLoading: true })
+
+    commitMutation(this.props.relay.environment, {
+      onCompleted: () => this.createBidderPosition(),
+      onError: e => console.error(e, e.message),
+      mutation: creditCardMutation,
+      variables: {
+        input: {
+          token: this.state.creditCardToken.tokenId,
+        },
+      },
+    })
+  }
+
+  createBidderPosition() {
+    commitMutation(this.props.relay.environment, {
+      onCompleted: (results, errors) => this.verifyBidPosition(results, errors),
+      onError: e => console.error(e, e.message),
+      mutation: bidderPositionMutation,
+      variables: {
+        input: {
+          sale_id: this.props.sale_artwork.sale.id,
+          artwork_id: this.props.sale_artwork.artwork.id,
+          max_bid_amount_cents: this.props.bid.cents,
+        },
+      },
+    })
+  }
+
+  verifyBidPosition(results: any, errors: PayloadError[] | null | undefined) {
+    const status = results.createBidderPosition.result.status
+
+    if (!errors && status === "SUCCESS") {
+      this.bidPlacedSuccessfully(results)
+    } else {
+      const message_header = results.createBidderPosition.result.message_header
+      const message_description_md = results.createBidderPosition.result.message_description_md
+      this.showBidResult(false, status, message_header, message_description_md)
+    }
+  }
+
+  @track({
+    action_type: Schema.ActionTypes.Success,
+    action_name: Schema.ActionNames.BidFlowPlaceBid,
+  })
+  bidPlacedSuccessfully(results) {
+    const positionId = results.createBidderPosition.result.position.id
+    this.queryForBidPosition(positionId).then(this.checkBidPosition.bind(this))
+  }
+
+  queryForBidPosition(bidderPositionID: string) {
+    const query = `
+        {
+          me {
+            bidder_position(id: "${bidderPositionID}") {
+              status
+              message_header
+              message_description_md
+              position {
+                id
+                processed_at
+                is_active
+                suggested_next_bid {
+                  cents
+                  display
+                }
+              }
+            }
+          }
+        }
+      `
+    return metaphysics({ query })
+  }
+
+  checkBidPosition(result) {
+    const bidderPosition = result.data.me.bidder_position.position
+    const status = result.data.me.bidder_position.status
+    if (status === "WINNING") {
+      this.showBidResult(true, "WINNING")
+    } else if (status === "PENDING") {
+      if (this.pollCount > MAX_POLL_ATTEMPTS) {
+        const md = `We're receiving a high volume of traffic and your bid is still processing.  \
```

Comment:
#minor but I believe we'll want the curly quote here.

Reply:
This will be addressed once we merge the `<ConfirmBid>` and `ConfirmFirstTimeBid` components.